l1-contracts/contracts/state-transition/ChainTypeManager.sol

```diff
@@ -125,6 +125,11 @@ contract ChainTypeManager is IChainTypeManager, ReentrancyGuard, Ownable2StepUpg
         return IZKChain(getZKChain(_chainId)).getAdmin();
     }
 
+    /// @notice Returns address of the RollupDAManager of the ZK Chain.
+    function getRollupDAManager(uint256 _chainId) external view returns (address) {
```

Comment:
Could you please not add the getter to CTM? (to keep the code smaller) On CTM the only method of such kind that we have is `getChainAdmin`, we have it just because it is a very common pattern to allow something only to the admin of a certain chain (ValidatorTimelock, ServerNotifier, etc all them need it), but almost noone will need to query rollup da manager from CTM.